charts/seed-bootstrap/charts/loki/templates/logging-service.yaml

```diff
@@ -0,0 +1,43 @@
+# Logging service duplicates the current loki service with the purpose of
+# providing the same target in fluent-bit outputs
+# Once the service is made available on all clusters (clusters are reconciled on daily basis) the
+# fluent-bits output will be switched (nickytd: the change shall be in a different PR )
+# After the switch the fluent-bit will deliver logs to the same target logging solving the problem
+# loki to vali migration. Since the target in fluent-bit is the same, fluent-bit will deliver logs to both
+# migrates and non migrated shoots.
```

Comment:
I think this description should not be in the service file, but rather in the commit message. It will be outdated soon with the following migration steps.
Referencing https://github.com/gardener/gardener/issues/7585 might make more sense.

Reply:
Thanks. The feedback is applied.